assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/AudienceItems.js

```diff
@@ -214,6 +220,49 @@ export default function AudienceItems( { savedItemSlugs = [] } ) {
 		?.filter( ( { name } ) => ! savedItemSlugs.includes( name ) )
 		.reduce( audiencesListReducer, {} );
 
+	const newBadgesToActivate = useSelect( ( select ) => {
+		if ( undefined === availableAudiences ) {
+			return undefined;
+		}
+
+		const { hasFinishedResolution, hasExpirableItem } = select( CORE_USER );
+
+		if ( ! hasFinishedResolution( 'getExpirableItems' ) ) {
+			return undefined;
+		}
+
+		return availableAudiences
+			.filter( ( { audienceType, name } ) => {
+				// Only activate if it is not a default audience, the badge is
+				// new and the user is viewing it for the first time.
+				return (
+					'DEFAULT_AUDIENCE' !== audienceType &&
+					! hasExpirableItem(
+						`${ AUDIENCE_ITEM_NEW_BADGE_SLUG_PREFIX }${ name }`
+					)
+				);
+			} )
+			.map( ( { name } ) => {
+				return `${ AUDIENCE_ITEM_NEW_BADGE_SLUG_PREFIX }${ name }`;
+			} );
+	} );
+
+	// Set an expiry for the new badges so that it is no longer shown after
+	// a certain time.
+	// `useDeepCompareEffect` is used here instead of `useEffect` because
+	// the array reference of `newBadgesToActivate` is not reliable and can
+	// cause the effect to unexpectedly run multiple times.
+	useDeepCompareEffect( () => {
+		if ( newBadgesToActivate !== undefined && newBadgesToActivate.length ) {
+			setExpirableItemTimers(
+				newBadgesToActivate.map( ( slug ) => ( {
+					slug,
+					expiresInSeconds: WEEK_IN_SECONDS * 4,
+				} ) )
+			);
```

Comment:
Setting expiry for the badges here instead of the individual audience items so that we don't have to make one HTTP request for each audience.